src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+                ObjectInputStream listInput = new ObjectInputStream(reader);
+                try {
+
+                    toDoList = (ArrayList<Task>) listInput.readObject();
+                    listInput.close();
+                } catch (ClassNotFoundException e) {
+                    System.out.println("class not found");
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        // Initialisation message
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("enter a command\n" +
+                "use command 'help' to see list of commands");
+
+        // Scan for user input
+        String userInput = sc.nextLine();
+        // Split user input into individual words
+        String[] split = userInput.split(" ");
+
+        while (!(userInput.equals("bye"))) {
+            if (userInput.equals("list")) {
+                for (int i = 0; i < toDoList.size(); i++) {
+                    Task t = toDoList.get(i);
+                    System.out.printf("%d. " + t.toString(), i + 1);
+                }
+            } else if (split[0].equals("help")) {
+                System.out.println("try the following commands:\n"
+                        + "   list (list out all tasks in your todo list)\n"
+                        + "   todo <task> (add a basic task to your todo list)\n"
+                        + "   deadline <task> /by <date> (add a task with a deadline to your todo list)\n"
+                        + "   event <task> /at <date> (add a new event with its corresponding date to your todo list)\n"
+                        + "   mark <task index> (mark that specific task as completed)\n"
+                        + "   unmark <task index> (remove mark from specific task)\n"
+                        + "   delete <task index> (remove task from todo list)\n"
+                        + "   bye (close application)");
+            } else if (split[0].equals("unmark")) {
+                try {
+                    int item = Integer.parseInt(split[1]);
+                    try {
+                        Task t = toDoList.get(item - 1);
+                        t.unmark();
+                        System.out.printf("Boo! more work to do: %s\n", t.getName());
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println(("the index you have entered does not exist!"));
+                    }
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("unmark description cannot be empty!");
+                } catch (NumberFormatException e) {
+                    System.out.println("unmark description must be an integer!");
+                }
+
+
```

Comment:
Might want to delete these empty lines too so that the formatting is abit more consistent